plugins/modules/cloudflare_dns.py

```diff
@@ -826,6 +876,10 @@ def ensure_dns_record(self, **kwargs):
                     do_update = True
             if (params['type'] == 'CNAME') and (cur_record['content'] != new_record['content']):
                 do_update = True
+            if cur_record['comment'] != new_record['comment']:
+                do_update = True
+            if sorted(cur_record['tags']) != sorted(new_record['tags']):
```

Comment:
No particular preference, just a note that this could be also achieved by:
```suggestion
            if set(cur_record['tags']) != set(new_record['tags']):
```